packages/presentation/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "@spinnaker/presentation",
+  "description": "Provides re-usable UI components for deck",
+  "version": "0.0.1",
+  "module": "dist/index.js",
+  "typings": "dist/index.d.ts",
+  "license": "Apache-2.0",
+  "scripts": {
+    "build": "rollup -c",
+    "clean": "npx shx rm -rf dist",
+    "prepublishOnly": "npm run build",
+    "watch": "rollup -c -w"
+  },
+  "peerDependencies": {
+    "lodash": "^4.17.20",
+    "react": "~16.14.0",
+    "react-dom": "^16.14.0"
```

Comment:
Wondering if we really need `react-dom` in this package?
@vigneshm

Reply:
Good call. I don't think we need it. I'll remove it for now and add it back when needed.